romanisim/parameters.py

```diff
@@ -84,6 +84,15 @@
     "linearity": None,
     "readnoise": 5.0 * u.DN,
     "saturation": 55000 * u.DN,
+    "photom": {"F062": 0.28798833 * u.MJy / u.sr,
+               "F087": 0.3911463 * u.MJy / u.sr,
+               "F106": 0.37039083 * u.MJy / u.sr,
+               "F129": 0.36495176 * u.MJy / u.sr,
+               "F146": 0.11750617 * u.MJy / u.sr,
+               "F158": 0.35352992 * u.MJy / u.sr,
+               "F184": 0.53612772 * u.MJy / u.sr,
+               "F213": 0.55763922 * u.MJy / u.sr,
+               }
```

Comment:
I missed where this is used.

Reply:
Sorry, this is used in other L3 code and came along for the ride here.

Reply:
And conceptually, this is the number of MJy / sr that is equal to 1 count / s in a fiducial L2 pixel?  Hopefully that's the same as (count / s / pix) / Jy * 10^6 Jy / MJy * pix / (0.1 / 60 / 60 * 180 / pi)^2 = abflux / 3631 * 10^6 Jy / MJy * pix / (0.1 / 60 / 60 * 180 / pi)^2?